indigo_app/views/tasks.py

```diff
@@ -90,6 +105,8 @@ def get_context_data(self, **kwargs):
         context['total_tasks'] = self.get_base_queryset().count()
 
         context["taxonomy_toc"] = TaxonomyTopic.get_toc_tree(self.request.GET)
```

Comment:
We no longer need the `all_topics` support, which was only used for the task view, so we can drop that parameter completely and remove the support, rather than defaulting it to True.